autoextract_poet/pages.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+from web_poet.pages import ItemPage
+
+from autoextract_poet.mixins import AutoExtractResponseShortcutsMixin
+from autoextract_poet.inputs import (
+    AutoExtractProductResponseData,
+    AutoExtractProductListResponseData,
+)
+
+
+@dataclass
+class AutoExtractProductPage(ItemPage, AutoExtractResponseShortcutsMixin):
+    """Generic product page."""
+
+    autoextract_response: AutoExtractProductResponseData
```

Comment:
In regular page objects we don't have access to the response structure.
So, maybe, to keep a consistent interface, we should only have access to `AutoExtractProductResponseData.data` in the page object.
Not that it is a problem, though

Reply:
This was addressed and now we have `data`.